recipe/setup_compiler.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+if [[ ! -d  $SRC_DIR/cf-compilers ]]; then
+    conda create -p $SRC_DIR/cf-compilers -c conda-forge --yes --quiet \
+      "binutils_impl_${build_platform}" \
+      "gcc_impl_${build_platform}" \
+      "gxx_impl_${build_platform}" \
+      "gfortran_impl_${build_platform}" \
+      "binutils_impl_${target_platform}=${binutils_version}" \
+      "gcc_impl_${target_platform}" \
+      "gxx_impl_${target_platform}" \
+      "gfortran_impl_${target_platform}" \
+      "sysroot_${target_platform}=${c_stdlib_version}"
```

Comment:
For consistency with CBC, you should use `sysroot_version` here.
```suggestion
      "${sysroot_name}_${target_platform}=${sysroot_version}"
```

Or we rewrite CBC so that `sysroot_{name,version}` become `c_stdlib{,_version}` (but given the current impossibility to extend an existing zip in conda-build, the former is definitely easier).

Reply:
Both resolve to
```
sysroot_linux-64=2.12
sysroot_linux-aarch64=2.17
[etc.]
```
Could you explain how they're different conceptually?

Reply:
Yes, for now, they do. `$sysroot_name` is for `cross_target_platform`. `c_stdlib` is for `target_platform`.

Reply:
I'll rename for better clarity